substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/doc-files/ReflectionConfigurationFilesHelp.txt

```diff
@@ -1,7 +1,8 @@
 One or several (comma-separated) paths to JSON files that specify which program elements should be made available via reflection.
+
 The JSON object schema is described at:
 
-   https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/reflect-config-schema-v1.0.0.json
+   https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/reflect-config-schema-v1.1.0.json
```

Comment:
This implies that `reflect-config.json` in JDK 21 now supports `typeReached` and `type`, and that `name` is deprecated.

However, `LegacyReflectionConfigurationParser` seems to still only support (and require) `name`.

Additionally, the use of `typeReached` seems to alwys result in a failure with an error in JDK 21, see https://github.com/loicottet/graalvm-community-jdk21u/blob/11f3306d6010aadefd044825c7e92151dc0a7ab8/substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/configure/ConfigurationParser.java#L239

So we should stick to v1.0.0

```suggestion
   https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/assets/reflect-config-schema-v1.0.0.json
```

Reply:
Good catch, those `v1.1.0` files should indeed be deleted